PathFinder.cc

```diff
@@ -64,33 +73,31 @@ void reset() {
   frontiers.clear();
   waypointResult.clear();
 
-  // // pure realtime, no any cache
-  // voxels.clear();
-  // voxelo.clear();
+  voxels.clear();

```

Comment:
If it's an instance you can just construct new vector and don't need to clear them like this.

Reply:
https://github.com/webaverse/app-wasm/pull/35/commits/769532ffed71e53e698347843b8e60034186d37a
Cleaned `reset()`, because of now create new `PathFinder` every time.